docs/src/main/asciidoc/stork-kubernetes.adoc

```diff
@@ -0,0 +1,479 @@
+      http:
+        paths:
+          - backend:
+              service:
+                name: color-service
+                port:
+                  name: http
+            path: /
+            pathType: Prefix
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    app.quarkus.io/commit-id: f747f359406bedfb1a39c57392a5b5a9eaefec56
+    app.quarkus.io/build-timestamp: 2022-03-31 - 10:36:56 +0000
+  labels:
+    color: blue
+    type: color-service
+    app.kubernetes.io/name: blue-service
+    app.kubernetes.io/version: "1.0"
+  name: blue-service
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: blue-service
+      app.kubernetes.io/version: "1.0"
+  template:
+    metadata:
+      annotations:
+        app.quarkus.io/commit-id: f747f359406bedfb1a39c57392a5b5a9eaefec56
+        app.quarkus.io/build-timestamp: 2022-03-31 - 10:36:56 +0000
+      labels:
+        color: blue
+        type: color-service
+        app.kubernetes.io/name: blue-service
+        app.kubernetes.io/version: "1.0"
+    spec:
+      containers:
+        - env:
+            - name: KUBERNETES_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          image: quay.io/quarkus/blue-service:1.0
+          imagePullPolicy: Always
+          name: blue-service
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    app.quarkus.io/commit-id: 27be03414510f776ca70d70d859b33e134570443
+    app.quarkus.io/build-timestamp: 2022-03-31 - 10:38:54 +0000
+  labels:
+    color: red
+    type: color-service
+    app.kubernetes.io/version: "1.0"
+    app.kubernetes.io/name: red-service
+  name: red-service
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/version: "1.0"
+      app.kubernetes.io/name: red-service
+  template:
+    metadata:
+      annotations:
+        app.quarkus.io/commit-id: 27be03414510f776ca70d70d859b33e134570443
+        app.quarkus.io/build-timestamp: 2022-03-31 - 10:38:54 +0000
+      labels:
+        color: red
+        type: color-service
+        app.kubernetes.io/version: "1.0"
+        app.kubernetes.io/name: red-service
+    spec:
+      containers:
+        - env:
+            - name: KUBERNETES_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          image: quay.io/quarkus/red-service:1.0
+          imagePullPolicy: Always
+          name: red-service
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+
+----
+Note that, as showed by the architecture diagram above, we will have a `color-service` backed by two Pods which are instances of the Red and Blue services. Moreover, the `color-service` is accessible from the outside of the cluster thanks to the Ingress resource.
```

Comment:
Just a thought, maybe it would be nicer to use the `<1>`, etc, comments in the code with explanations, as for the rest client reactive guide here: https://quarkus.io/guides/rest-client-reactive#create-the-jax-rs-resource